src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $gapBetween;
+
+  padding-inline: 10px;
+  width: ($cellSize + $gapBetween) * $columns;
+
+  @each $dayOfWeek, $number in $week {
+    &--start-day--#{$dayOfWeek} {
+      .calendar__day--1 {
+        margin-left: ($cellSize + $gapBetween) * $number - $cellSize;
```

Comment:
It's work well, I think, it's SASS because.
Change for:
```
      .calendar__day--1 {
        margin-left: calc((#{$cellSize} + #{$gapBetween}) * #{$number} - #{$cellSize});
      }
```